test/data/low_income_thresholds.test.ts

```diff
@@ -52,3 +52,22 @@ test('low-income thresholds are equivalent in JSON config and incentives', async
     }
   }
 });
+
+test('low-income thresholds have HH sizes 1-8', async t => {
+  const hasRequiredKeys = (obj: Record<string, unknown>) => {
+    const keys = Object.keys(obj);
+    return [1, 2, 3, 4, 5, 6, 7, 8].every(num => keys.includes(num.toString()));
```

Comment:
Question: I noticed some thresholds go up to ten. Do we ever use those values in the API? I'm mostly curious about this for data modeling purposes in the admin app.

Reply:
Values above 8 aren't ever used because the `household_size` request param is limited to >= 1 and <= 8. But we put the higher numbers in the data when the original source had them, just in case.

Extra info:

I think the request param is limited to 8 because the official AMI numbers (which were relevant for the IRA incentives) were only published up to HH size = 8.

In the case of AMI for IRA incentives, there's a simple formula to extend the thresholds to bigger HH sizes. But I don't think there generally is such a formula for all the various state and utility thresholds. And a lot of those only give thresholds up to 8, so I think we're generally stuck with 8 as the upper limit.